Tests/Algorithm/AlgorithmTradingTests.cs

```diff
@@ -91,7 +102,7 @@ public void SetHoldings_ZeroToShort(decimal leverage)
         public void SetHoldings_ZeroToShort_SmallConstantFeeStructure(decimal leverage)
         {
             Security msft;
-            var algo = GetAlgorithm(out msft, 1, 1);
+            var algo = GetAlgorithm(out msft, leverage, 1);
```

Comment:
I added the multiple leverage test cases, because in previous versions of the code I got different results with different leverages. In these test methods, answers are manually computed and checked, they just happen to (correctly) give the same answer for different values of the leverage input value. I didn't see an easier way to avoid proliferation of test methods. :)